docs/system/process-metrics.md

```diff
@@ -20,32 +20,179 @@ metrics](runtime-environment-metrics.md).
+<!-- endsemconv -->
+
+### Metric: `process.memory.usage`
+
+<!-- semconv metric.process.memory.usage(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `process.memory.usage` | UpDownCounter | `By` | The amount of physical memory in use. |
+<!-- endsemconv -->
+
+<!-- semconv metric.process.memory.usage(full) -->
+<!-- endsemconv -->
+
+### Metric: `process.memory.virtual`
+
+<!-- semconv metric.process.memory.virtual(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `process.memory.virtual` | UpDownCounter | `By` | The amount of committed virtual memory. |
+<!-- endsemconv -->
+
+<!-- semconv metric.process.memory.virtual(full) -->
+<!-- endsemconv -->
+
+### Metric: `process.disk.io`
+
+<!-- semconv metric.process.disk.io(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `process.disk.io` | Counter | `By` | Disk bytes transferred. |
+<!-- endsemconv -->
+
+<!-- semconv metric.process.disk.io(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `process.disk.process.disk.io.direction` | string | The direction of the data transfer for this data point. | `read` | Recommended |
+
+`process.disk.process.disk.io.direction` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `read` | read |
+| `write` | write |
+<!-- endsemconv -->
+
+### Metric: `process.network.io`
+
+<!-- semconv metric.process.network.io(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `process.network.io` | Counter | `By` | Network bytes transferred. |
+<!-- endsemconv -->
+
+<!-- semconv metric.process.network.io(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `process.network.io.direction` | string | The direction of the data transfer for this data point. | `receive` | Recommended |
+
+`process.network.io.direction` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `receive` | receive |
+| `transmit` | transmit |
+<!-- endsemconv -->
+
+### Metric: `process.threads`
+
+<!-- semconv metric.process.threads(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `process.threads` | UpDownCounter | `{thread}` | Process threads count. |
+<!-- endsemconv -->
+
+<!-- semconv metric.process.threads(full) -->
+<!-- endsemconv -->
+
```

Comment:
We are missing `process.open_file_descriptors`

Reply:
Resolved in [5ad3a95](https://github.com/open-telemetry/semantic-conventions/pull/330/commits/5ad3a95ab2ddaf681fef56fbe757d1dedda68744)